source/includes/examples/tf-example-push-logs-to-s3.rst

```diff
@@ -0,0 +1,38 @@
+.. code-block:: 
+   :copyable: true 
+
+   resource "mongodbatlas_project" "project_tf" {
+     name   = var.atlas_project_name
```

Comment:
`var.atlas_project_name` or `var.project_name` ? Same later

Reply:
Removed as this example has been removed from the page.